bliss/encoder/encoder.py

```diff
@@ -80,14 +81,15 @@ def __init__(
         self.double_detect = double_detect
         self.use_checkerboard = use_checkerboard
 
+        num_downsample = int(np.log2(self.tile_slen)) - 1
```

Comment:
Would you assert that `tile_size` is a power of 2 here (or somewhere)?

Also, probably a good idea to round the log of the tile size to the nearest integer before truncating, in case of slight floating point imprecision

Reply:
Done